ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/ParticipantStateIntegrationSpecBase.scala

```diff
@@ -679,7 +679,10 @@ object ParticipantStateIntegrationSpecBase {
 
   private val darReader = DarReader { case (_, is) => Try(DamlLf.Archive.parseFrom(is)) }
   private val archives =
-    darReader.readArchiveFromFile(new File(rlocation("ledger/test-common/Test-stable.dar"))).get.all
+    darReader
+      .readArchiveFromFile(new File(rlocation("ledger/test-common/model-tests.dar")))
```

Comment:
Might read more nicely (and live on fewer lines) if you pull out `new File(…)` into a variable.